### ISSUE TITLE
CLI doesn't apply recommended rules correctly

### ISSUE THREAD

**Issue by jackiehluo** [ember-template-lint/ember-template-lint#556]
I've been running the CLI on a few files (we're moving off `ember-cli-template lint` and using this package directly), and it doesn't appear to be linting correctly! We have the default recommended rules turned on, but I deliberately broke a few of them to check the command, and they're getting unevenly applied.

For instance, this code:

```
<SectionHeader
    @title='Item' @secondaryButtonText="New Item"
    @secondaryButtonAction={{action "show"}} />
```

...doesn't trigger any errors even though it should because all of the line breaks are off, and it should enforce "Item" to be double-quoted.

This code, however, in the very same file, does trigger an error:

```
{{{item.description}}}
```

```
21:14  error  Usage of triple curly brackets is unsafe  no-triple-curlies
```

Any idea what might be happening? Thank you so much!

**Comment by jackiehluo** [comment]
The `.template-lintrc.js` is just:

```
'use strict';

module.exports = {
  extends: 'recommended'
};
```

Command I'm running:
```
./node_modules/.bin/ember-template-lint app/templates/components/items-section.hbs
```

And we're on the latest version of `ember-template-lint`, `0.8.23`.

**Comment by rwjblue** [comment]
I believe you need to be on the 1.0.0-beta.x versions to get support for linting angle bracket invocation. Would you mind updating and testing?

**Comment by jackiehluo** [comment]
Ahhhhh, that's 100% what happened. Sorry, sorry, my bad! Didn't realize it was a beta version (I think it was the Ember 3.4 upgrade that added it to our project).
